src/core/system.conf.in

```diff
@@ -27,6 +27,7 @@
 #RuntimeWatchdogSec=0
 #ShutdownWatchdogSec=10min
 #CapabilityBoundingSet=
+#NoNewPrivileges=false
```

Comment:
one last nitpick, i am noticing only now: while true/false, 1/0, on/off and yes/no are all accepted for our boolean options, we generally only use "yes" and "no" in our default configuration files, just as as an act of being systematic.

Could you change false → no hence? Sorry for noticing this only so late.

Looks good otherwise.